catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/metacard/QueryMetacardApplication.java

```diff
@@ -183,7 +197,7 @@ String getSubjectIdentifier() {
     return subjectIdentity.getUniqueIdentifier(SecurityUtils.getSubject());
   }
 
-  private int getOrDefaultParam(Request request, String key, int defaultValue) {
+  private static int getOrDefaultParam(Request request, String key, int defaultValue) {
```

Comment:
❓ Just curious why the change to static here, why not the function below as well?

Reply:
Fair point. I've made the other one `static` as well.